.github/workflows/test_precommit.yml

```diff
@@ -45,11 +45,11 @@ jobs:
     - name: Check style with black
       run: |
         source venv/bin/activate
-        black --check .
+        black --exclude model_api/python/openvino/__init__.py --check .
```

Comment:
Can we add skip directives to the file? Passing ignore list to command line is not very convenient and scalable

Reply:
black reads `pyproject.toml`. But having `pyproject.toml` would suggest that we support building a package from it